src/content/developers/tutorials/logging-events-smart-contracts/index.md

```diff
@@ -12,6 +12,8 @@ sourceUrl: https://ethereumdev.io/logging-data-with-events/
 address: "0x19dE91Af973F404EDF5B4c093983a7c6E3EC8ccE"
 ---
 
+# Logging data from smart contracts with events {#logging-data-from-smart-contracts-with-events}
```

Comment:
Why do we need to add a header to the beginning of tutorials? (Is the lack thereof causing issues with the translation support?) If it is possible I'd much rather remove leading headers elsewhere rather than adding them here.

If we do need to add headers we should probably use H2s as [we already have a H1 on all tutorial pages](https://github.com/ethereum/ethereum-org-website/blob/994a03c747ced8022b7064050fd68b83fa670381/src/templates/tutorial.js#L177) that we pull from the title in the frontmatter.